src/Files.App/Helpers/ShareItemHelpers.cs

```diff
@@ -19,15 +21,32 @@ public static bool IsItemShareable(ListedItem item)
 				(!item.IsShortcut || item.IsLinkItem) &&
 				(item.PrimaryItemAttribute != StorageItemTypes.Folder || item.IsArchive);
 
-		public static void ShareItems(IEnumerable<ListedItem> itemsToShare)
+		public static async void ShareItems(IEnumerable<ListedItem> itemsToShare)
```

Comment:
```suggestion
		public static async Task ShareItemsAsync(IEnumerable<ListedItem> itemsToShare)
```
And the task that this returns should be the return value in `ShareItemAction.cs`

Reply:
👍